api/v1beta1/types_class.go

```diff
@@ -133,7 +128,7 @@ func (sc *SubnetClassSpec) setDefaults(cidr string) {
 }
 
 // setDefaults sets default values for SecurityGroupClass.
-func (sgc *SecurityGroupClass) setDefaults(dir SecurityRuleDirection) {
+func (sgc *SecurityGroupClass) setDefaults(dir SecurityRuleDirection) { //nolint:unparam
```

Comment:
The linter complains because it is always called with the exact same argument values and with this PR it looks like we crossed the limit of how many times we can call the function with the same arguments before the linter complains.